### PR TITLE
add mesos client tests #182

### PR DESCRIPTION
**Pull request by asnaedae**

### CODE REVIEW

File: tests/spec/mesos_slave/mesos_slave_spec.rb [hunk 1]
```diff
@@ -4,6 +4,30 @@
   it { should be_installed }
```
Comment (tayzlor) [inline tests/spec/mesos_slave/mesos_slave_spec.rb]: should we add a test here to ensure the master service is not running on the slave?

Comment (asnaedae) [inline tests/spec/mesos_slave/mesos_slave_spec.rb]: good plan.

File: tests/spec/mesos_slave/mesos_slave_spec.rb [hunk 2]
```diff
@@ -4,6 +4,30 @@
...
+    it            { should be_file }
+    its(:content) { should match /^MASTER=`cat \/etc\/mesos\/zk`$/ }
+  end
+
+  describe file('/etc/mesos-slave') do
+    it { should be_directory }
+  end
+  describe file('/etc/mesos-slave/work_dir') do
+    it            { should be_file }
+    its(:content) { should contain '/tmp/mesos' }
```
Comment (wallies) [inline tests/spec/mesos_slave/mesos_slave_spec.rb]: i dont know whether this will always be /tmp/mesos, on the other hand we should just test for defaults

Comment (tayzlor) [inline tests/spec/mesos_slave/mesos_slave_spec.rb]: yeah will break if someone configures it differently but should be ok within the confines of a test harness

Comment (asnaedae) [inline tests/spec/mesos_slave/mesos_slave_spec.rb]: will strip it out for now, doesn't really add a huge amount of confidence by being there.

**Pull request merged**